.travis.yml

```diff
@@ -3,5 +3,7 @@ scala:
   - 2.11.6
 services:
   - mongodb
+jdk:
+  - oraclejdk8
```

Comment:
Does the WDL parser require java 8?  We are pulling java 7 into the docker image.